finance-admin-api/awsclient/awsclient.go

```diff
@@ -0,0 +1,48 @@
+package awsclient
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"os"
+)
+
+type AWSClient interface {
+	PutObject(ctx context.Context, params *s3.PutObjectInput, optFns ...func(*s3.Options)) (*s3.PutObjectOutput, error)
+	Options() s3.Options
+}
+
+func NewClient(ctx context.Context) (AWSClient, error) {
+	awsRegion, ok := os.LookupEnv("AWS_REGION")
+
+	if !ok || awsRegion == "" {
+		awsRegion = "eu-west-1" // default region
```

Comment:
Does this need to be passed in as well just in caes the region might change in the infra, I think that it will than just pull in the new one 🤔